.env.local

```diff
@@ -0,0 +1,2 @@
+MAL_USERNAME=
```

Comment:
This file needs the other parameters in the `.env` file living on the c9 instance